web/installer

```diff
@@ -11,7 +11,7 @@
  * file that was distributed with this source code.
  */
 
-process($argv);
+process( is_array($argv) ? $argv : array() );
```

Comment:
I see, those 2 spaces. *mumbles something about being counter-productive* I'll fix and submit tonight. Should I make it a commit on this same pr, or better to make a fresh pr?